client/components/shared/NewBankForm.js

```diff
@@ -179,12 +193,11 @@ export default class NewBankForm extends React.Component {
                         <span className={ `option-legend fa fa-${this.state.expanded ?
                           'minus' : 'plus'}-circle` }
```

Comment:
should be square, to be coherent with search operations. (or change to circle in search operations)